designs/virtualization/4.3/vm-details/dedicated-resources/dedicated-resources.md

```diff
@@ -0,0 +1,27 @@
+---
+parent: Virtualization
+version: 4.4
+CNV-version: 2.3
+---
+
+# VM details - dedicated-resources
+
+### note
+The original design was CPU Pinning. It changed to workload Isolation, now it is Dedicated Resources.
+
+## In-page placement
+![VM details - Overview tab](img/in-page-0-1.jpg)
+Within the VM details page > Details tab > Overview section we added the “Dedicated resource” field.
+The default state is “No dedicated resources applied”.
+
+![VM details - Overview tab](img/in-page-0-2.jpg)
+When activated the field will display "Dedicated resources applied"
+
+## Edit
+![Modal - default](img/modal-0-1.jpg)
+Click on the edit pencil will pop up this modal.
+This modal includes only one checkbox. Checking it will “schedule this workload with dedicated resources (guaranteed policy)”.
+A supporting text exposes to the user that this feature is available only on nodes that contain this node label: cpumanager=true
+
+![Modal - no nodes found](img/modal-0-2.jpg)
+In case such a node cannot be found, a warning notification will present itself, mentioning there no such nodes. The reason we allowed to create dedicated resources where there are no nodes is that it is a general stand we took that we do allow the user to create policies that will end up in scheduling failure, in case that user wants to achieve some scheduling policy state and want the workload to fail in case that policy’s requirements cannot be met. Another reason is to allow the user to create policies that will cause scheduling failure and fix the problem later from another policy.
```

Comment:
Should there be a design from the details page showing dedicated resources will be used but no resources match the criteria currently? I think it might be important for the user to know incase they think they've created such nodes but might be missing a label and this would let them confirm.

Reply:
Good use case!
We're trying to tackle that and similar cases in the upcoming Scheduling section. I will share with you that design soon. We need your help with this :)